src/components/deals/details/ReportDealButton.tsx

```diff
@@ -0,0 +1,31 @@
+'use client'
+import React, { useState } from 'react'
+import { CiFlag1 } from 'react-icons/ci'
+import ReportDealPopup from './ReportDealPopup'
+
+interface ReportDealButtonProps {
+  dealId: string
+}
+
+export default function ReportDealButton({ dealId }: ReportDealButtonProps) {
+  const [popupOpen, setPopupOpen] = useState<boolean>(false)
+
+  const handleClick = async () => {
+    setPopupOpen(true)
+  }
+  return (
+    <>
+      <button
+        onClick={handleClick}
+        className="flex items-center justify-center gap-x-2 rounded-md border-2 border-red-50 bg-red-50 px-4 py-2 text-center text-red-500 transition-colors hover:border-red-300 "
```

Comment:
Your contrast is not high enough here as it is currently 3.44 (https://www.siegemedia.com/contrast-ratio#%23ef4444-on-%23fef2f2). 

Try using `text-red-700` (https://www.siegemedia.com/contrast-ratio#%23b91c1c-on-%23fef2f2)

Reply:
NICE! Fixed in latest commit